CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Ensure that non-ISPF reports do not get a value set for `is_oda` (for historical reasons, the app relies on non-ISPF reports and activities not having an `is_oda` value set)
 - Validate that the value of `is_oda` is set for ISPF reports
 - Reorder the fields on the new report form per design
+- show the if a report contains ODA or non-ODA on the report view
```

Comment:
Would you mind removing the stray "the" and adding that this only affects ISPF reports, for consistency with the line below?